.buildkite/scripts/setenv.sh

```diff
@@ -41,10 +56,14 @@ exportVars() {
   fi
 }
 
-
-if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats-metricbeat" ]]; then
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats-metricbeat" || "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-metricbeat" ]]; then
   exportVars
   export RACE_DETECTOR="true"
   export TEST_COVERAGE="true"
   export DOCKER_PULL="0"
+  export TEST_TAGS="oracle"
+fi
+
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-metricbeat" ]]; then
```

Comment:
Can you document what's the reason for this particular condition and variable?

Reply:
To help with the reviews, I'd suggest adding a comment in the discussion with the link to the commit. For instance, in this case, the discussion is still open, but https://github.com/elastic/beats/pull/38081/commits/fd09867e41f75354b00c4b89f2aabd9e0b7f76a1 added a comment, hence answering the question.

Otherwise, reviews can be tedious to be done.  🙏